libs/sdk-backend-tiger/tests/integrated/catalog.test.ts

```diff
@@ -16,6 +17,7 @@ describe("tiger catalog", () => {
         const result = await backend.workspace(testWorkspace()).catalog().load();
```

Comment:
`typeof result === IWorkspaceCatalog`

`IWorkspaceCatalog` declare in https://github.com/gooddata/gooddata-ui-sdk/blob/a2edbaacb37433eb981186d0f2ddada5faec52b7/libs/sdk-backend-spi/src/workspace/ldm/catalog.ts#L155

So when convert types `IWorkspaceCatalog` to JSON-Schema will get no properties like this

```
{
  "$schema": "http://json-schema.org/draft-07/schema#",
  "$ref": "#/definitions/IWorkspaceCatalog",
  "definitions": {
    "IWorkspaceCatalog": {
      "type": "object",
      "additionalProperties": false,
      "properties": {},
      "description": "Instance of workspace catalog with loaded items"
    }
  }
}
```